### PR TITLE
Added Diablo 2 saves

### PR DESCRIPTION
**Pull request by FalsePhilosopher**
I tried to follow the format of the existing library as best I could as what the export function spit out was slightly different from the readme, but hopefully this saves somebody else two weeks of their life hahaha.

### CODE REVIEW

**Comment by FalsePhilosopher** [comment]
Also thanks for everything you do for the PS scene, mad love from over here bud!

**Comment by FalsePhilosopher** [comment]
I didn't fully read the readme, my bad haha.

File: PS4/CUSA18937/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## PS4 Saves - 18937
+
+| Icon | Filename | Description |
+|------|----------|-------------|
+| ![Diablo® II: Resurrected™](icon0.png) | [00000001.zip](00000001.zip){: .btn .btn-purple } | Storage-runes/gems |
+| ![Diablo® II: Resurrected™](icon0.png) | [00000002.zip](00000001.zip){: .btn .btn-purple } | kk-lvl 40 lightning sorceress |
+| ![Diablo® II: Resurrected™](icon0.png) | [00000003.zip](00000001.zip){: .btn .btn-purple } | datboi-lvl 30 barbarian |
+| ![Diablo® II: Resurrected™](icon0.png) | [00000004.zip](00000001.zip){: .btn .btn-purple } | Boots-lvl 88 paladin-already beaten hell |
+| ![Diablo® II: Resurrected™](icon0.png) | [00000005.zip](00000001.zip){: .btn .btn-purple } | assassassin-lvl 15 assassin |
```
Comment (bucanero) [inline PS4/CUSA18937/README.md:17]:
```suggestion
| ![Diablo® II: Resurrected™](icon0.png) | [00000001.zip](00000001.zip){: .btn .btn-purple } | Storage-runes/gems |
| ![Diablo® II: Resurrected™](icon0.png) | [00000002.zip](00000002.zip){: .btn .btn-purple } | kk-lvl 40 lightning sorceress |
| ![Diablo® II: Resurrected™](icon0.png) | [00000003.zip](00000003.zip){: .btn .btn-purple } | datboi-lvl 30 barbarian |
| ![Diablo® II: Resurrected™](icon0.png) | [00000004.zip](00000004.zip){: .btn .btn-purple } | Boots-lvl 88 paladin-already beaten hell |
| ![Diablo® II: Resurrected™](icon0.png) | [00000005.zip](00000005.zip){: .btn .btn-purple } | assassassin-lvl 15 assassin |
```

File: PS4/CUSA18937/saves.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+00000001.zip=[CUSA18937] Diablo® II: Resurrected™ - Storage-runes/gems
+00000002.zip=[CUSA18937] Diablo® II: Resurrected™ - kk-lvl 40 lightning sorceress
+00000003.zip=[CUSA18937] Diablo® II: Resurrected™ - datboi-lvl 30 barbarian
+00000004.zip=[CUSA18937] Diablo® II: Resurrected™ - Boots-lvl 88 paladin-already beaten hell
+00000005.zip=[CUSA18937] Diablo® II: Resurrected™ - assassassin-lvl 15 assassin
```
Comment (bucanero) [inline PS4/CUSA18937/saves.txt:5]:
```suggestion
00000001.zip=Storage-runes/gems
00000002.zip=kk-lvl 40 lightning sorceress
00000003.zip=datboi-lvl 30 barbarian
00000004.zip=Boots-lvl 88 paladin-already beaten hell
00000005.zip=assassassin-lvl 15 assassin
```

**Review by bucanero — APPROVED** [review]

**Comment by bucanero** [comment]
> I tried to follow the format of the existing library as best I could as what the export function spit out was slightly different from the readme, but hopefully this saves somebody else two weeks of their life hahaha.

the PR was fine, I just did some slight adjustments so it could be browsed fine from Apollo

thanks for sharing your saves 👍

**Pull request merged**